Classes/DataProcessing/FilesProcessor.php

```diff
@@ -219,7 +219,7 @@ protected function processFiles(array $dimensions = []): ?array
             } else {
                 $data[$key] = $this->getFileUtility()->processFile($fileObject, $dimensions, $cropVariant);
 
-                $cropVariants = json_decode($fileObject->getProperty('crop'), true);
+                $cropVariants = json_decode((string)$fileObject->getProperty('crop'), true);
```

Comment:
I think it will better to check if crop property is not null & then try to decode & handle cropVariants